tests/shakedown/shakedown/clients/authentication.py

```diff
@@ -123,3 +126,41 @@ def __init__(self, token):
     def __call__(self, r):
         r.headers['Authorization'] = "token={}".format(self.token)
         return r
+
+
+# TODO(karsten): This is a verbatim copy of ..dcos.command.run_dcos_command to avoid circular imports.
+def run_dcos_command(command, raise_on_error=False, print_output=True):
```

Comment:
That's a good point. I think we'll have to reorganize a lot of packages for that to happen, eg `shakedown.dcos.command.*` includes commands for agents and the CLI. I'd much rather have a `shakedown.resource.Agent.run_command` and a `shakedown.resource.CLI.run_command`. This would be similar to Boto3 for AWS. However, I don't think I can do this in this PR.

Reply:
I've found a way. It was not too complex. Thanks for raising the issue!